pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -134,7 +134,18 @@ def mirrored_publish(version):
         version: The repository version to mirror-publish
     """
     with RpmPublication.create(version, pass_through=False) as publication:
-        for (result, relative_path) in metadata_files_for_mirroring[str(version.repository.pk)]:
+        repo_metadata_files = metadata_files_for_mirroring[str(version.repository.pk)]
+
+        has_repomd_signature = "repodata/repomd.xml.asc" in repo_metadata_files.keys()
+        has_sqlite = any([".sqlite" in href for href in repo_metadata_files.keys()])
+
+        publication.package_checksum_type = CHECKSUM_TYPES.UNKNOWN
+        publication.metadata_checksum_type = CHECKSUM_TYPES.UNKNOWN
+        publication.gpgcheck = 0
+        publication.repo_gpgcheck = has_repomd_signature
+        publication.sqlite_metadata = has_sqlite
```

Comment:
Could you remind me why we need those to be set at all in the mirror case? I presume they doi not change the outcome.

A redmine issue describing the problem this PR solves will be helpful. I'm not sure what fails or which error happens without this fix.

Reply:
They don't change the outcome, and there is no error or failure, but it would be better if they weren't blatantly wrong.

Reply:
It's a small thing and inconsequential, I'd just rather our records weren't completely wrong

Reply:
Ok, I do not have a strong opinion on that. 
If you prefer to have it, so be it.